warehouse/ingest-core/src/main/java/datawave/ingest/table/config/MetadataTableConfigHelper.java

```diff
@@ -58,6 +60,26 @@ private String setFrequencyCombiner(TableOperations tops, String scopeName) thro
         return stem;
     }
 
+    // Add the SummingCombiner to the indexed column.
+    private String setIndexCombiner(TableOperations tops, String scopeName) throws AccumuloException, TableNotFoundException, AccumuloSecurityException {
+        String stem = String.format("%s%s.%s", Property.TABLE_ITERATOR_PREFIX, scopeName, "IndexCombiner");
+        setPropertyIfNecessary(tableName, stem, "11," + SummingCombiner.class.getName(), tops, log);
+        setPropertyIfNecessary(tableName, stem + ".opt.columns", ColumnFamilyConstants.COLF_I.toString(), tops, log);
+        setPropertyIfNecessary(tableName, stem + ".opt.lossy", "true", tops, log);
+        setPropertyIfNecessary(tableName, stem + ".opt.type", "VARLEN", tops, log);
+        return stem;
+    }
+
+    // Add the SummingCombiner to the reverse indexed column.
+    private String setReverseIndexCombiner(TableOperations tops, String scopeName) throws AccumuloException, TableNotFoundException, AccumuloSecurityException {
+        String stem = String.format("%s%s.%s", Property.TABLE_ITERATOR_PREFIX, scopeName, "ReverseIndexCombiner");
+        setPropertyIfNecessary(tableName, stem, "12," + SummingCombiner.class.getName(), tops, log);
+        setPropertyIfNecessary(tableName, stem + ".opt.columns", ColumnFamilyConstants.COLF_I.toString(), tops, log);
```

Comment:
This should be the COLF_RI column family.

Reply:
Good catch! Fixed.